release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -138,6 +138,21 @@ You can now move etcd from a root volume (Cinder) to a dedicated ephemeral local
 
 //For more information, see < insert link to Deploying OpenStack with rootVolume and etcd on local disk when docs merge>.
 
+[id="ocp-4-15-agent-vsphere-credentials"]
+==== Configure vSphere integration with the Agent-based Installer
+
+You can now configure your cluster to use vSphere while creating the `install-config.yaml` file for an Agent-based Installation.
+For more information, see xref:../installing/installing_with_agent_based_installer/installation-config-parameters-agent.adoc#installation-configuration-parameters-additional-vsphere_installation-config-parameters-agent[Additional VMware vSphere configuration parameters].
+
+[id="ocp-4-15-agent-baremetal-configuration"]
+==== Additional bare metal configurations during Agent-based installation
+
+You can now make additional configurations for the bare metal platform while creating the `install-config.yaml` file for an Agent-based Installation.
+These new options include host configuration, network configuration, and baseboard management controller (BMC) details.
```

Comment:
Quick question: are `host configuration, network configuration, and BMC` all fields in a configuration? If so, I _think_ these would need backticks.

Reply:
Good question! I meant these terms in a general sense and not to refer to specific fields. And I checked the parameters themselves just now. There is a field for `hosts` and `bmc`, but nothing specifically for network.

 Since I mean these terms in a general sense and not to refer to any specific fields, can I keep it this way?

Reply:
I _think_ it is ok as is. Referring to them as `fields` in the following sentence makes it seem like you are referring to specific fields if that makes sense? Again, no need to change.